### PR TITLE
update tikv config for pitr

### PR DESCRIPTION
**Pull request by joccau**
Signed-off-by: joccau <zak.zhao@pingcap.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -1585,6 +1585,46 @@ Raft Engine 相关的配置项。
 + 在集群资源占用率较高的情况下，是否允许 BR 自动限制备份使用的资源，减少对集群的影响。详情见[自动调节](/br/br-auto-tune.md)。
 + 默认值：true
 
+## log-backup
+
+用于日志备份相关的配置项
```
Comment (ran-huang) [inline tikv-configuration-file.md:1590]:
```suggestion
用于日志备份相关的配置项。
```

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -1585,6 +1585,46 @@ Raft Engine 相关的配置项。
...
+
+### `enable`
+
++ 用于开启日志备份功能。
++ 默认值：false
+
+### `file-size-limit`
+
++ 日志备份任务备份的数据达到一定大小时，自动 flush 到外部存储中。
```
Comment (ran-huang) [inline tikv-configuration-file.md:1599]:
```suggestion
+ 日志备份任务的备份数据达到一定大小时，自动 flush 到外部存储中。
```

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -1585,6 +1585,46 @@ Raft Engine 相关的配置项。
...
++ 默认值：256MB
```
Comment (ran-huang) [inline tikv-configuration-file.md:1600]: 有无取值范围限制

File: tikv-configuration-file.md [hunk 4]
```diff
@@ -1585,6 +1585,46 @@ Raft Engine 相关的配置项。
...
+
+### `initial-scan-pending-memory-quota`
+
++ 日志备份服务在扫描增量数据时，用于存放扫描数据的缓存大小
```
Comment (ran-huang) [inline tikv-configuration-file.md:1604]:
```suggestion
+ 日志备份任务在扫描增量数据时，用于存放扫描数据的缓存大小。
```

File: tikv-configuration-file.md [hunk 5]
```diff
@@ -1585,6 +1585,46 @@ Raft Engine 相关的配置项。
...
++ 默认值：取 TOTAL_MEM * 0.1 与 512MB 中的最小值
```
Comment (ran-huang) [inline tikv-configuration-file.md:1605]: TOTAL_MEM 是指什么，BR 节点的内存？

File: tikv-configuration-file.md [hunk 6]
```diff
@@ -1585,6 +1585,46 @@ Raft Engine 相关的配置项。
...
+
+### `initial-scan-rate-limit`
+
++ 日志备份服务在扫描增量数据时，用于扫描时吞吐限流参数
```
Comment (ran-huang) [inline tikv-configuration-file.md:1609]:
```suggestion
+ 日志备份任务在扫描增量数据时，用于扫描时吞吐限流的参数。
```

File: tikv-configuration-file.md [hunk 7]
```diff
@@ -1585,6 +1585,46 @@ Raft Engine 相关的配置项。
...
++ 默认值：60，即默认限流 60 MB/s
+
+### `max-flush-interval`
+
++ 日志备份任务将备份数据写入到 External Storage 的最大间隔时间
```
Comment (ran-huang) [inline tikv-configuration-file.md:1614]:
```suggestion
+ 日志备份任务将备份数据写入到外部存储的最大间隔时间。
```

File: tikv-configuration-file.md [hunk 8]
```diff
@@ -1585,6 +1585,46 @@ Raft Engine 相关的配置项。
...
++ 默认值：5min
+
+### `num-threads`
+
++ 日志备份功能占用的线程数亩
```
Comment (ran-huang) [inline tikv-configuration-file.md:1619]:
```suggestion
+ 日志备份功能占用的线程数目。
```

File: tikv-configuration-file.md [hunk 9]
```diff
@@ -1585,6 +1585,46 @@ Raft Engine 相关的配置项。
...
++ 默认值：CPU * 0.5，
++ 可调整范围：[2, 12]
+
+### `temp-path`
+
++ 日志文件存放的临时目录，日志文件预先先写入临时目录，然后 flush 到外部存储中
++ 默认值：${deploy-dir}/data/log-backup-temp
```
Comment (ran-huang) [inline tikv-configuration-file.md:1626]:
```suggestion
+ 默认值：`${deploy-dir}/data/log-backup-temp`
```

File: tikv-configuration-file.md [hunk 10]
```diff
@@ -1585,6 +1585,46 @@ Raft Engine 相关的配置项。
...
+
+### `num-threads`
+
++ 日志备份功能占用的线程数亩
++ 默认值：CPU * 0.5，
++ 可调整范围：[2, 12]
+
+### `temp-path`
+
++ 日志文件存放的临时目录，日志文件预先先写入临时目录，然后 flush 到外部存储中
```
Comment (ran-huang) [inline tikv-configuration-file.md:1625]:
```suggestion
+ 日志文件存放的临时目录，日志文件预先写入临时目录，然后 flush 到外部存储中。
```

File: tikv-configuration-file.md (hunk 3 above)
Comment (joccau) [reply tikv-configuration-file.md:1600]: 没有

File: tikv-configuration-file.md (hunk 5 above)
Comment (joccau) [reply tikv-configuration-file.md:1605]: 整个机器的内存大小

File: tikv-configuration-file.md [hunk 11]
```diff
@@ -1585,6 +1585,46 @@ Raft Engine 相关的配置项。
...
+
+### `max-flush-interval`
+
++ 日志备份任务将备份数据写入到外部存储的最大间隔时间。
++ 默认值：5min
+
+### `num-threads`
+
++ 日志备份功能占用的线程数目。
++ 默认值：CPU * 0.5，
```
Comment (shichun-0415) [inline tikv-configuration-file.md:1620]:
```suggestion
+ 默认值：CPU * 0.5
```

File: tikv-configuration-file.md [hunk 12]
```diff
@@ -1585,6 +1585,46 @@ Raft Engine 相关的配置项。
...
+
+### `file-size-limit`
+
++ 日志备份任务的备份数据达到一定大小时，自动 flush 到外部存储中。
++ 默认值：256MB
+
+### `initial-scan-pending-memory-quota`
+
++ 日志备份任务在扫描增量数据时，用于存放扫描数据的缓存大小。
++ 默认值：取 TOTAL_MEM * 0.1 与 512MB 中的最小值
```
Comment (shichun-0415) [inline tikv-configuration-file.md:1605]:
```suggestion
+ 默认值：取 TOTAL_MEM * 0.1 与 512 MB 中的较小值
```

File: tikv-configuration-file.md [hunk 13]
```diff
@@ -1585,6 +1585,46 @@ Raft Engine 相关的配置项。
...
+
+### `initial-scan-rate-limit`
+
++ 日志备份任务在扫描增量数据时，用于扫描时吞吐限流的参数。
```
Comment (shichun-0415) [inline tikv-configuration-file.md:1609]:
```suggestion
+ 日志备份任务在扫描增量数据时的吞吐限流参数。
```

**Review by shichun-0415 — APPROVED** [review]
Rest LGTM

**Review by fubinzh — APPROVED** [review]
LGTM

File: tikv-configuration-file.md (hunk 5 above)
Comment (ran-huang) [reply tikv-configuration-file.md:1605]: 这个应该需要说明一下

**Comment by IANTHEREAL** [comment]
LGTM

File: tikv-configuration-file.md [hunk 14]
```diff
@@ -1585,6 +1585,46 @@ Raft Engine 相关的配置项。
...
+
+### `file-size-limit`
+
++ 日志备份任务的备份数据达到一定大小时，自动 flush 到外部存储中。
++ 默认值：256MB
+
+### `initial-scan-pending-memory-quota`
+
++ 日志备份任务在扫描增量数据时，用于存放扫描数据的缓存大小。
++ 默认值：取 TOTAL_MEM * 0.1 与 512 MB 中的较小值
```
Comment (ran-huang) [inline tikv-configuration-file.md:1605]:
```suggestion
+ 默认值：`min(TOTAL_MEM * 0.1, 512 MB)`，TOTAL_MEM 指整个机器的内存大小。
```

**Review by ran-huang — APPROVED** [review]

File: br/pitr-usage.md [hunk 15]
```diff
@@ -23,7 +23,7 @@ summary: 了解如何使用 PiTR。
 | TiDB | 8 核+ | 16 GB+ | SAS | c5.2xlarge | 2 |
```
Comment (ran-huang) [inline br/pitr-usage.md:23]: This file is aligned in https://github.com/pingcap/docs/pull/9823/commits/0b175c28c93723603b3bcf2956793bd84f5da335

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**